twitter_nlp/test/test_nlp.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import nltk
+from nltk.corpus import twitter_samples
+from nltk.tag import pos_tag_sents
+
+from twitter_nlp.core import nlp
+
+class TestSum(unittest.TestCase):
+    """Testing unit to determine if output remains the same.
+
+    Basic unittest to determine if the outputs from the counting 
+    function (simple_nlp) still match the original results.
+    """
+    def test_compute(self):
+        """Tests whether output of class matches the orignal ones."""
+        obj = nlp()
+        obj.simple_nlp()
+        self.assertAlmostEqual(
```

Comment:
What kind of behaviour are you testing with this unit test? A test to check if the functionality does change is not really what unit tests should do. You should try to test for a specific intention/behaviour to verify that the programming is working as intended.

It also does not allow for any improvements or alterations to be made to the software, since you only compare if the output is the same as a previous run.

Also, if a fellow programmer alters your code and this test fails, how would they know what is wrong and how to fix it?